src/cookbook/design/themes.md

```diff
@@ -151,19 +208,30 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: appName,
       theme: ThemeData(
-        // Define the default brightness and colors.
-        brightness: Brightness.dark,
-        primaryColor: Colors.lightBlue[800],
+        useMaterial3: true,
 
-        // Define the default font family.
-        fontFamily: 'Georgia',
+        // Define the default brightness and colors.
+        colorScheme: ColorScheme.fromSwatch(
+          primarySwatch: Colors.purple,
+          // TRY THIS: Change to "Brightness.light"
+          //           and see that all colors change
+          //           to better contrast a light background
+          brightness: Brightness.dark,
+        ),
 
         // Define the default `TextTheme`. Use this to specify the default
         // text styling for headlines, titles, bodies of text, and more.
-        textTheme: const TextTheme(
-          displayLarge: TextStyle(fontSize: 72, fontWeight: FontWeight.bold),
-          titleLarge: TextStyle(fontSize: 36, fontStyle: FontStyle.italic),
-          bodyMedium: TextStyle(fontSize: 14, fontFamily: 'Hind'),
+        textTheme: TextTheme(
+          displayLarge:
+              const TextStyle(fontSize: 72, fontWeight: FontWeight.bold),
+          titleLarge:
+              GoogleFonts.oswald(fontSize: 30, fontStyle: FontStyle.italic),
+          bodyMedium: GoogleFonts.merriweather(),
+          // TRY THIS: Change one of the GoogleFonts
+          //           to "lato", "poppins", or "lora"
+          //           The title uses "titleLarge"
+          //           and the middle text uses "bodyMedium"
```

Comment:
I'd move this up one line, above the `bodyMedium` line that it applies to.  Also, should it still refer to `titleLarge`, or should it revert to `displaySmall` here?

Reply:
**reply:** I changed it to match the code. All is well.